### PR TITLE
Alert dismiss with key

### LINKED ISSUE #2140: Add support for custom action handlers for links in the alerts
For more info check #2124

### PR DESCRIPTION
**Pull request by stanleychh**
Origin draft PR #2135 

Fixes #2140

### CODE REVIEW

File: client/luigi-client.d.ts [hunk 1]
```diff
@@ -136,6 +136,7 @@ export declare interface UxManager {
    * @param {Object} settings.links.LINK_KEY object containing the data for a particular link. To properly render the link in the alert message refer to the description of the **settings.text** parameter
    * @param {string} settings.links.LINK_KEY.text text which replaces the link identifier in the alert content
    * @param {string} settings.links.LINK_KEY.url url to navigate when you click the link. Currently, only internal links are supported in the form of relative or absolute paths
+   * @param {Object} settings.links.LINK_KEY.dismissKey dismissKey which represents the key of link.
```
Comment (alexandra-simeonova) [inline client/luigi-client.d.ts:139]:
```suggestion
   * @param {Object} settings.links.LINK_KEY.dismissKey dismissKey which represents the key of the link.
```

**Review by hardl — CHANGES_REQUESTED** [review]
codewise looks good, but documentation is missing

**Comment by stanleychh** [comment]
@hardl Thanks for your time for reviewing. I will add the documentation.

**Review by hardl — APPROVED** [review]
LGTM

**Pull request merged**